src/test/java/ClassTest.java

```diff
@@ -27,8 +27,8 @@ public TestClass(String y) {
 	public void subclass() {
 		ReflectClass<?> clazz = ReflectClass.wrap(ClassTest.class);
 
-		assertEquals("TestClass", clazz.getSubclass("TestClass").getName());
-		assertEquals("StaticTestClass", clazz.getSubclass("StaticTestClass").getName());
+		assertEquals("TestClass", clazz.getSubclass("ClassTest$TestClass").getSimpleName());
```

Comment:
There should be no need for the prefix. We should rather change the implementation so that it works without it because it is counter intuitive to have to add the name of the parent to the child's name